frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -1263,10 +1263,9 @@ export const funnelLogic = kea<funnelLogicType>({
                 openPersonsModal({
                     url: converted ? step.converted_people_url : step.dropped_people_url,
                     title: funnelTitle({
-                        step: converted ? step.order : -step.order,
-                        breakdown_value: breakdownValues.isEmpty
-                            ? undefined
-                            : breakdownValues.breakdown_value.join(', '),
+                        converted,
+                        // Note - when in a legend the stepIndex is used instead
+                        step: typeof stepIndex === 'number' ? stepIndex + 1 : step.order + 1,
```

Comment:
Why so?

Reply:
I'm not sure but its something to do with the series stuff. It passes in the "baseline" step which for somereason always has an `.order` of 0. So the stepIndex was the cleanest workaround I could find

Reply:
Improved the comment (although I still don't know why it is this way...)